planning/behavior_path_planner/autoware_behavior_path_planner_common/src/utils/drivable_area_expansion/drivable_area_expansion.cpp

```diff
@@ -360,6 +362,43 @@ void calculate_expansion_distances(
   }
 }
 
+void add_bound_point(std::vector<Point> & bound, const Pose & pose)
+{
+  const auto p = convert_point(pose.position);
+  PointDistance nearest_projection;
+  nearest_projection.distance = std::numeric_limits<double>::infinity();
+  size_t nearest_idx = 0UL;
+  for (auto i = 0UL; i + 1 < bound.size(); ++i) {
+    const auto prev_p = convert_point(bound[i]);
+    const auto next_p = convert_point(bound[i + 1]);
+    const auto projection = point_to_segment_projection(p, prev_p, next_p);
+    if (projection.distance < nearest_projection.distance) {
+      nearest_projection = projection;
+      nearest_idx = i;
+    }
+  }
+  Point new_point;
+  new_point.x = nearest_projection.point.x();
+  new_point.y = nearest_projection.point.y();
+  new_point.z = bound[nearest_idx].z;
+  if (
+    universe_utils::calcDistance2d(new_point, bound[nearest_idx]) > 0.1 &&
+    universe_utils::calcDistance2d(new_point, bound[nearest_idx + 1]) > 0.1) {
+    bound.insert(bound.begin() + nearest_idx + 1, new_point);
+  }
+}
+
+void add_bound_points(
+  std::vector<Point> & left_bound, std::vector<Point> & right_bound,
+  const std::vector<Pose> & path_poses, const std::vector<double> & curvatures)
```

Comment:
curvatures is not used anywhere?

Reply:
Thank you, I removed it: [d76d3b7](https://github.com/autowarefoundation/autoware.universe/pull/8136/commits/d76d3b7b50d3c662be9af4e74d81da12062c5587)